.github/workflows/rust.yml

```diff
@@ -33,6 +33,7 @@ jobs:
       - run: cargo test -p dicom-pixeldata --features gdcm
       # test dicom-pixeldata without default features
       - run: cargo test -p dicom-pixeldata --no-default-features
+      - run: cargo test -p dicom-ul --features async
```

Comment:
`async` was enabled by default, so we were not testing the code with `async` disabled. I fixed the situation by feature-gating. 

I also favored making the feature opt-in, as it facilitates decoupling the crate from the tokio ecosystem when async isn't needed, and does not prevent other crates in DICOM-rs from enabling it. This is a similar approach to `dicom-pixeldata` enabling features from `dicom-transfer-syntax-registry`.